migrations/900000000000042_add_comment_to_providers.up.sql

```diff
@@ -0,0 +1,6 @@
+-- SQL migration to add an optional comment field to providers
+BEGIN;
+
+ALTER TABLE mergestat.providers ADD COLUMN comment TEXT;
```

Comment:
```suggestion
-- SQL migration to add an optional description field to providers
BEGIN;

ALTER TABLE mergestat.providers ADD COLUMN description TEXT;
```

Reply:
@riyaz-ali any reason for the comment vs description?

Reply:
no particular reason 😅 happy to rename it to `description` 👍